src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -724,6 +725,7 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
         pass_config->disable<ngraph::pass::AddFakeQuantizeFusion>();
         // TransposeReduction can be enabled when Transpose-Conv-Transpose patterns will be handled in ngraph transformations
         pass_config->disable<ngraph::pass::TransposeReduction>();
+        pass_config->disable<ngraph::pass::ConcatReduceFusion>();
```

Comment:
If some plugin wants to disable or parametrize transformation which is located inside MOC so we have to keep this transformation in CommonOptimizations because MOC pipelines is used by MO also.

Also please describe why GNA disable this fusion.

Reply:
@GlebKazantaev Problem is in `NopElimination` transformation. It breaks GNA plugin. It is the first reason. The second reason is that we don't need `ConcatReduceFusion` transformation at all because operations from patterns are not supported.

Reply:
@nkogteva Thanks for explanation! I added comments with reasons why we have to disable this transformation